docs/blueprints/EP036.rst

```diff
@@ -60,49 +78,61 @@ The system of multi-tables will be replicated accross all switches. For now we w
     |   goto_table: 1  |         |   goto_table: 2  |         |   goto_table: 3  |         |   goto_table: 5  |         |                  |         |                  |
     +------------------+         +------------------+         +------------------+         +------------------+         +------------------+         +------------------+
 
-New collection, **tables**
-==========================
+New collection, **pipelines**
+=============================
+
+There a couple of rules for a proper pipeline:
 
-Initially proposed here is a datastructured representing a linked list (without cycles)
+  - The table number for the next table should be always higher than the current one. This number is found as the `table_id` value in the dictionary ``{"instruction_type": "goto_table", "table_id": 1}``
+  - The pipeline represents a linked list (no loops). A miss flow entry should be present in every table except in the end of the linked list which represents the end on the list.
+
+Initially proposed here is a datastructured representing a linked list. 
 
   .. code:: javascript
 
     "pipeline": {
-      0: {
-        "table_miss_flow": {
-          "priority": 123, "match": {},
-          "instructions": [{"instruction_type": "goto_table", "table_id": 1}]
-        },
-        "napps_table_groups": {"coloring": ["base"],"of_lldp": ["base"]}
-      },
-      1: {
-        "table_miss_flow": {
-          "priority": 123, "match": {},
-          "instructions": [{"instruction_type": "goto_table","table_id": 2}]
-        },
-        "napps_table_groups": {"flow_manager": []}
-      },
-      2: {
-        "table_miss_flow": {
-          "priority": 123, "match": {},
-          "instructions": [{"instruction_type": "goto_table","table_id": 3}]
-        },
-        "napps_table_groups": {"mef_eline": ["evpl"]}
-      },
-      3: {
-        "table_miss_flow": {
-          "priority": 123, "match": {},
-          "instructions": [{"instruction_type": "goto_table","table_id": 5}]
-        },
-        "napps_table_groups": {"mef_eline": ["epl"]}
-      }
-      4: {
-        "table_miss_flow": {},
-        "napps_table_groups": {"INT": []}
-      }
-      5: {
-        "table_miss_flow": {},
-        "napps_table_groups": {}
+      {
+        "id": "9cba7e27a33c45",

```

Comment:
It might be work checking with @italovalcy or @jab1982, if they'd like a "description: Optional[str]" field to add optional (helpful) description about the pipeline. If one day network engineers end up with more pipeline, having a meaningful description can be handy to remember which pipeline is for what purpose. Wdyt, can you also confirm with them?